packages/core/src/payment/strategies/amazon-pay-v2/amazon-pay-v2-payment-processor.ts

```diff
@@ -123,6 +123,16 @@ export default class AmazonPayV2PaymentProcessor {
                           publicKeyId,
                       },
                   };
+
+            return total
+                ? {
+                      ...buttonOptions,
+                      estimatedOrderAmount: {
+                          amount: total.toString(),
+                          currencyCode,
```

Comment:
We we need the cart currency here? Since that's the transactional currency where as the storeConfig is for display purpose.

Reply:
You're right, thanks!